doc/releases/changelog-dev.md

```diff
@@ -21,11 +21,14 @@
     Rydberg atoms.
   * A new user-facing `rydberg_interaction` function is added, which returns a `RydbergHamiltonian` containing
```

Comment:
Not sure what happened, but the changelog seems to have been reverted and has references to `RydbergHamiltonian`.